Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs

```diff
@@ -283,14 +276,15 @@ public void UpdateController(OVRHand hand, OVRSkeleton ovrSkeleton, Transform tr
             }
         }
 
-#if LATER
+        //used to track the input that was last raised
```

Comment:
To fix code validation

```suggestion
        // Used to track the input that was last raised
```